build.yaml

```diff
@@ -1977,15 +1979,7 @@ steps:
      tar xvf wheel-container.tar
      python3 -m pip install --no-dependencies hail-*-py3-none-any.whl
      hailctl config set batch/bucket hail-test-dmk9z
-     cat > test.py <<EOF
-     import hail as hl
-     t = hl.utils.range_table(50, 3)
-     t = t.filter((t.idx % 3 == 0) | ((t.idx / 7) % 3 == 0))
-     n = t.count()
-     print(f'n {n}')
-     assert n == 17
-     EOF
-     HAIL_BILLING_PROJECT=test HAIL_DONT_RETRY_500=1 HAIL_QUERY_BACKEND=service python3 test.py
+     HAIL_BILLING_PROJECT=test HAIL_DONT_RETRY_500=1 HAIL_QUERY_BACKEND=service python3 -m pytest --log-cli-level=INFO -s -vv --instafail --durations=50 /io/test/
```

Comment:
I think you forgot to check in the test file

Reply:
>.<